typescript/src/entities/user.ts

```diff
@@ -95,120 +96,128 @@ export class User extends Entity {
   public systemRoles?: Array<SystemRole>;
 
-  @User.property({arrayType: "EmailAddress"})
+  @User.property({ arrayType: "EmailAddress" })
   public emailAddresses?: Array<EmailAddress>;
 
-  @User.property({arrayType: "Notification"})
+  @User.property({ arrayType: "Notification" })
   public notifications?: Array<Notification>;
 
-  @User.property({arrayType: "Notification"})
+  @User.property({ arrayType: "Notification" })
   public sentNotifications?: Array<Notification>;
 
-  @User.property({arrayType: "Assignment"})
+  @User.property({ arrayType: "Assignment" })
   public assignments?: Array<Assignment>;
 
-  @User.property({arrayType: "UserCompany"})
+  @User.property({ arrayType: "UserCompany" })
   public userCompanies?: Array<UserCompany>;
 
-  @User.property({arrayType: "Draft"})
+  @User.property({ arrayType: "Draft" })
   public drafts?: Array<Draft>;
 
-  @User.property({arrayType: "CompanyInvitation"})
+  @User.property({ arrayType: "CompanyInvitation" })
   public companyInvitations?: Array<CompanyInvitation>;
 
-  @User.property({arrayType: "Address"})
+  @User.property({ arrayType: "Address" })
   public addresses?: Array<Address>;
 
-  @User.property({arrayType: "MerchiFile"})
+  @User.property({ arrayType: "MerchiFile" })
   public uploadFiles?: Array<MerchiFile>;
 
-  @User.property({arrayType: "JobComment"})
+  @User.property({ arrayType: "JobComment" })
   public forwardedJobComments?: Array<JobComment>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public appliedJobs?: Array<Job>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public managedJobs?: Array<Job>;
 
-  @User.property({arrayType: "Job"})
+  @User.property({ arrayType: "Job" })
   public draftingJobs?: Array<Job>;
 
-  @User.property({arrayType: "Product"})
+  @User.property({ arrayType: "Product" })
   public saved_products?: Array<Product>;
 
-  @User.property({arrayType: "Cart"})
+  @User.property({ arrayType: "Cart" })
   public carts?: Array<Cart>;
 
-  @User.property({arrayType: "Payment"})
+  @User.property({ arrayType: "Payment" })
   public payments?: Array<Payment>;
 
-  @User.property({arrayType: "EnrolledDomain"})
+  @User.property({ arrayType: "EnrolledDomain" })
   public enrolledDomains?: Array<EnrolledDomain>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public responsibleInvoices?: Array<Invoice>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public createdInvoices?: Array<Invoice>;
 
-  @User.property({arrayType: "Invoice"})
+  @User.property({ arrayType: "Invoice" })
   public invoicesHas?: Array<Invoice>;
 
-  @User.property({arrayType: "DomainInvitation"})
+  @User.property({ arrayType: "DomainInvitation" })
   public sentDomainInvitations?: Array<DomainInvitation>;
 
-  @User.property({arrayType: "DomainInvitation"})
+  @User.property({ arrayType: "DomainInvitation" })
```

Comment:
this has nothing to do with times formatting.

Reply:
I am so sorry about this as this is propably comes from my auto syntax fixer plugin "ale > prettier", I was totally unaware of this as I set it fix things for me aggressively.

I am willing to a make another pr for syntax issue

however as this pr has already been polluted, I am afraid I do not spend time manually rollback what prettier has already done for time saving reason.